src/app/plan/plans/plans.component.ts

```diff
@@ -64,4 +67,14 @@ export class PlansComponent implements OnInit {
       }
     );
   }
+
+  DeleteClick(id: number): void {
+    this.dialogsService
+      .confirm('Confirm Dialog', 'Are you sure you want delete plan?')
```

Comment:
Are you sure you want to delete this plan?

Reply:
+